parser/parser_test.go

```diff
@@ -331,7 +331,19 @@ var parseTests = []parseTest{
 	{
 		"SimpleService",
 		"service TestService {\n  rpc Foo(In) returns (Out);\n}\n message In{} message Out{}",
-		`service { name: "TestService" method { name:"Foo" input_type:".In" output_type:".Out" } }` +
+		`service { name: "TestService" method { name:"Foo" input_type:".In" output_type:".Out" client_streaming: false server_streaming: false } }` +
```

Comment:
This shouldn't have changed. client_streaming/server_streaming should not have been set if they were false.